.github/workflows/release-latest.yml

```diff
@@ -28,6 +28,22 @@ jobs:
       - run: |
           git tag v${{ fromJson(steps.pkg.outputs.json).version }}
           git push origin v${{ fromJson(steps.pkg.outputs.json).version }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6 # 2.0.0
+        with:
+          install: true
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
+      - name: Build injection image and push to github packages
+        run: |
+          node scripts/set-version-for-init-image.js ${{ fromJson(steps.pkg.outputs.json).version }}
```

Comment:
Why is this needed?

Reply:
This changes the version in the `package.json` so that the correct version of the tracer is installed.

I could instead do this as a build arg in docker, which would prevent all of this process. Would you prefer that?

Reply:
I'm not sure what that would look like so hard to say. What would be the benefit of the current approach compared to the build arg?

Reply:
As it turns out this is way simpler than I had previously thought. In my next rev of this (gonna push in a few min) this won't need any build args, nor will it need this script. It will just pick up whatever tarball is around and use that.